cmd/gafFilter/gafFilter.go

```diff
@@ -0,0 +1,76 @@
+// Command Group:
+
+// Output a subset of a gaf file without "NOT" entries in qualifier field
```

Comment:
the usage statement is a little too specific you can just describe that it filters gaf entries, even if right now all if does is remove NOTs